acceptance/testdata/auth/auth-login-logout.txtar

```diff
@@ -0,0 +1,29 @@
+# We aren't logged in at the moment, but GH_HOST will override the
+# need to login. We are going to clear GH_HOST first to ensure no
+# overrides are happening
+
+# Copy $GH_TOKEN to a new env var
+setEnvVar LOGIN_TOKEN $GH_TOKEN
```

Comment:
I'll need to check this but I'm surprised that

```
env LOGIN_TOKEN $GH_TOKEN
```

Wouldn't work here.

Reply:
I'm getting this error

<img width="920" alt="Screenshot 2024-10-21 at 9 10 54 AM" src="https://github.com/user-attachments/assets/a8ad7773-f839-4d2f-b273-bc8214fbf989">

I'm not sure why it isn't working, either

Reply:
It looks like there may be an extra `=` at the end of `LOGIN_TOKEN`, based on the output, but I'd still expect that to match on `GH_TOKEN` because we're using regex... Also, notably, it hangs when I comment out the assertions on lines 7 & 8, so something weird is happening

Reply:
I think it needs to be `env LOGIN_TOKEN=$GH_TOKEN` sorry.

Reply:
Ah, that did it 👍 Good callout

Reply:
Now I can remove the two new shared functions 🥳